exercises/camicia/instructions.md

```diff
@@ -0,0 +1,83 @@
+# Instructions
+
+In this exercise, you will simulate a game very similar to the classic card game **Camicia**.
+Your program will receive the initial configuration of two players' decks and must simulate the game until it ends (or detect that it will never end).
+
+## Rules
+
+- The deck is split between **two players**.
+  The player's cards are read from left to right, where the leftmost card is the top of the deck.
+- A round consists of both players playing at least one card/turn.
+- Players take a turn by placing the **top card** of their deck onto a central pile.
+- If the card is a **number card** (2-10), play simply passes to the other player.
+- If the card is a **payment card**, a penalty must be paid:
+  - `J` → opponent must pay 1 card
+  - `Q` → opponent must pay 2 cards
+  - `K` → opponent must pay 3 cards
+  - `A` → opponent must pay 4 cards
+- If the player paying a penalty reveals another payment card, that player stops paying the penalty.
+  The other player must then pay a penalty based on the new payment card.
+- If the penalty is fully paid without interruption, the player who laid the **last payment card** collects the central pile and places it at the bottom of their deck.
+  That player then starts the next round.
+- The moment when a player collects cards from the central pile is called a **trick**.
+- If a player has all the cards in their possession after a trick, the game **ends**.
+- The game **ends** when one player has all the cards in their deck.
```

Comment:
```suggestion
- If a player has all the cards in their possession after a trick, the game **ends**.
```

These two lines seem to be saying the say thing. We should drop one or the other.